### ISSUE TITLE
请大佬建立个telegram群组

### ISSUE THREAD

**Issue by lq95v5** [Gamuxorg/AppImage-CN#3]
qq不支持linux，telegram方便些。。。

**Comment by probonopd** [comment]
检查弗朗茨。它有一个AppImage
![franz](https://user-images.githubusercontent.com/2480569/33514136-6ec05cd8-d74e-11e7-8517-215116fd63c6.png)
https://meetfranz.com

**Comment by shouhuanxiaoji** [comment]
build a telegram channel is very easy, but most of Chinese people cannot connect it without VPN.
Mr. probono is the founder of AppImage and maintainer. It have a offical IRC channel.

建一个TG频道很容易，但是大多数国人连不上，原因你懂得。
Mr. probono 就是AppImage项目的创始人和主要作者，你可以在IRC的官方#appimage频道找到需要的人。

你可以建立一个频道，然后拉人试试看，如果有一部分人愿意加入，我们会过去的。

**Comment by shouhuanxiaoji** [comment]
建好了，搜一下@gamuxzh就可以进入

**Comment by Andy2Clyde** [comment]
TG上不去，科学也不行。

**Comment by shouhuanxiaoji** [comment]
@Andy2Clyde  先到QQ群274328087

**Comment by Andy2Clyde** [comment]
Already in the QQ group.


发送自 Outlook<http://aka.ms/weboutlook>


________________________________
发件人: 小鸡 <notifications@github.com>
发送时间: 2017年12月5日 6:34
收件人: Gamuxorg/AppImage-CN
抄送: Andy2Clyde; Mention
主题: Re: [Gamuxorg/AppImage-CN] 请大佬建立个telegram群组 (#3)


@Andy2Clyde<https://github.com/andy2clyde> 先到QQ群274328087

―

**Comment by shouhuanxiaoji** [comment]
@probonopd hello, we have registed an BAIDU account for you. please check your email probono@puredarwin.org to receive

**Comment by probonopd** [comment]
Thank you @shouhuanxiaoji I am a bit busy right now

**Comment by shouhuanxiaoji** [comment]
@probonopd  
It doesn't matter. You're busy first. In addition, we find it a little wasteful to build a forum alone. Is it possible to incorporate bbs.appimage.cn into the official forum? Is it possible to open a Chinese plate in the official forum?

**Comment by probonopd** [comment]
You would like a Chinese sub-forum on https://discourse.appimage.org/? That would be no problem, just let me know and I will open it.

**Comment by shouhuanxiaoji** [comment]
@probonopd yes,then we will redirect bbs.appimage.cn to the chinese sub-forum of discourse.appimage.org.

**Comment by probonopd** [comment]
https://discourse.appimage.org/c/chinese

**Comment by shouhuanxiaoji** [comment]
@probonopd How did IPFS study? We have added two servers more, Beijing and Lithuania.
